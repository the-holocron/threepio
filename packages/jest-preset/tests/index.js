const config = require('../jest-preset.json');

describe('Preset', () => {
	describe('Jest', () => {
		it('should render the configuration', () => expect(config).toMatchSnapshot());
	});
});
