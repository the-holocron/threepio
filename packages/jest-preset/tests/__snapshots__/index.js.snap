// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preset Jest should render the configuration 1`] = `
Object {
  "collectCoverage": true,
  "collectCoverageFrom": Array [
    "<rootDir>/src/**/*.{js,jsx}",
    "!<rootDir>/**/{__mocks__, mocks}/**",
    "!<rootDir>/**/*.{mock,mocks}.js",
    "!<rootDir>/**/{__stories__, stories}/**",
    "!<rootDir>/**/{__tests__, tests}/__snapshots__/**",
  ],
  "coverageThreshold": Object {
    "global": Object {
      "branches": 80,
      "functions": 80,
      "lines": 80,
      "statements": 80,
    },
  },
  "modulePathIgnorePatterns": Array [
    "<rootDir>/dist/",
  ],
  "testRegex": Array [
    "(/__tests__/.*|(\\\\.|/)(test|spec))\\\\.[jt]sx?$",
    "(/tests/.*|(\\\\.|/)(test|spec))\\\\.[jt]sx?$",
  ],
  "transform": Object {
    "^.+\\\\.js$": "babel-jest",
  },
  "transformIgnorePatterns": Array [
    "<rootDir>/node_modules/",
  ],
}
`;
